tcf_website/static/course/javascript/instr_data.js

```diff
@@ -7,7 +7,7 @@
 jQuery(function($) {
     /* Fetch instructor data for given course */
     var pageSize = "1000";
-    var instrEndpoint = `/api/instructors/?course=${courseID}` +
+    var instrEndpoint = `/api/instructors/?course=${window.courseID}` +
```

Comment:
eslint complains about `courseID is undefined` if you don't prefix global variables with `window.`